modules/telco-core-scheduling.adoc

```diff
@@ -10,17 +10,28 @@ New in this release::
 * No reference design updates in this release
 
 Description::
-* The scheduler is a cluster-wide component responsible for selecting the right node for a given workload. It is a core part of the platform and does not require any specific configuration in the common deployment scenarios. However, there are few specific use cases described in the following section.
+The scheduler is a cluster-wide component responsible for selecting the right node for a given workload.
+It is a core part of the platform and does not require any specific configuration in the common deployment scenarios.
+However, there are few specific use cases described in the following section.
+
 NUMA-aware scheduling can be enabled through the NUMA Resources Operator.
 For more information, see "Scheduling NUMA-aware workloads".
 
 Limits and requirements::
-* The default scheduler does not understand the NUMA locality of workloads. It only knows about the sum of all free resources on a worker node. This might cause workloads to be rejected when scheduled to a node with the topology manager policy set to `single-numa-node` or `restricted`.
-** For example, consider a pod requesting 6 CPUs and being scheduled to an empty node that has 4 CPUs per NUMA node. The total allocatable capacity of the node is 8 CPUs and the scheduler will place the pod there. The node local admission will fail, however, as there are only 4 CPUs available in each of the NUMA nodes.
-** All clusters with multi-NUMA nodes are required to use the NUMA Resources Operator. Use the `machineConfigPoolSelector` field in the `KubeletConfig` CR to select all nodes where NUMA aligned scheduling is needed.
-* All machine config pools must have consistent hardware configuration for example all nodes are expected to have the same NUMA zone count.
+* The default scheduler does not understand the NUMA locality of workloads.
+It only knows about the sum of all free resources on a worker node.
+This might cause workloads to be rejected when scheduled to a node with the topology manager policy set to single-numa-node or restricted.
```

Comment:
??
```suggestion
This might cause workloads to be rejected when scheduled to a node with the topology manager policy set to `single-numa-node` or `restricted`.
```